gobreaker.go

```diff
@@ -189,6 +189,11 @@ func (cb *CircuitBreaker) State() State {
 	return state
 }
 
+// Returns internal counters
```

Comment:
The method comment should start with the method name `Counts`.
See https://blog.golang.org/godoc